edit: add information regarding the enable lookup on mongo views (#497)
mia-platform/documentation: docs/development_suite/api-console/api-design/mongo_views.md

MongoDB will run an [aggregation pipeline](https://www.mongodb.com/docs/manual/core/aggregation-pipeline/) starting from the `source` collection.    
When you create a new view, the console will set a default pipeline which returns only `PUBLIC` documents. You can edit it through the dedicated editor.   

> <!-- TODO: Change screenshot -->

Reviewer: has this been done?

Author: Yes. Removing it right now.

This feature is available from version `5.2.2` of [CRUD Service](/runtime_suite/crud-service/10_overview_and_usage.md). Creating a view while using a lower version of CRUD Service would make it crash at boot.
:::

Moreover, from version 11.5.0 of the Console you can also decide to enable lookups to get [Writable Views](/runtime_suite/crud-service/50_writable_views.md). To do that you have to enable the _Enable Lookup_ toggle included next to the `source` collection input.

:::caution

> To enable lookup, the CRUD Service installed in the project must be of version 6.9.0 or superior, as explicitely in the Docker Image of your service. If you're using a public variable to have different versions among environments, make sure every environment have version 6.9.0 or superior.

Reviewer: ```suggestion
To enable lookup, the CRUD Service installed in the project must be of version 6.9.0 or higher, as explicitly in the Docker Image of your service. If you are using a public variable to have different versions among environments, make sure every environment have version 6.9.0 or higher.
```
